crossdock/docker-compose.yml

```diff
@@ -35,6 +35,8 @@ services:
         image: jaegertracing/xdock-java
         ports:
             - "8080-8082"
+        depends_on:
+            - test_driver
```

Comment:
By default Java xdock uses UDP sender which is created straight away when the container is started. If the `test_driver` hasn't been started before java container will fail.